internal/controller/api/v1alpha1/installenv_types.go

```diff
@@ -83,6 +83,8 @@ type InstallEnvSpec struct {
 	// Future versions will allow for multiple ClusterDeployments and this reference will be
 	// removed.
 	ClusterRef *ClusterReference `json:"clusterRef"`
+	// Json formatted string containing the user overrides for the initial ignition config
+	IgnitionConfigOverride string `json:"ignitionConfigOverride"`
```

Comment:
How are the overrides applied.

Reply:
Good question, it is set as a filed in the DB, so every time you change it the old values are replaced, @carbonin correct me if i'm wrong.

Reply:
Right, the override is stored as json, but we're using the coreos ignition library to do the merge when we create the discovery iso

ref: https://github.com/openshift/assisted-service/blob/9c3609bc4336bb99d2888e7bc6c8f5a49316893b/internal/ignition/ignition.go#L946